src/Core/Content/Product/Aggregate/ProductConfiguratorSetting/ProductConfiguratorSettingEntity.php

```diff
@@ -17,68 +17,26 @@ class ProductConfiguratorSettingEntity extends Entity
     use EntityCustomFieldsTrait;
     use EntityIdTrait;
 
-    /**
-     * @var string
-     *
-     * @deprecated tag:v6.7.0 - Will be natively typed
-     */
-    protected $productId;
+    protected string $productId;
 
-    /**
-     * @var string
-     *
-     * @deprecated tag:v6.7.0 - Will be natively typed
-     */
-    protected $optionId;
+    protected string $optionId;
 
-    /**
-     * @var string|null
-     *
-     * @deprecated tag:v6.7.0 - Will be natively typed
-     */
-    protected $mediaId;
+    protected ?string $mediaId;
```

Comment:
you should add `null` as default values to nullable properties. before they were implicit `null`, but if you now access them without initializing, you will get an error. 

you can use this regex for search replace in PHPStorm: search string: `protected \?(.*) \$(.*)(?<!null);` 
replace string: `protected \?$1 \$$2 = null;`